resources/js/Pages/Maintenance.vue

```diff
@@ -1,189 +1,65 @@
 <script setup lang="ts">
-import { Head } from "@inertiajs/vue3";
-import { OhVueIcon } from "oh-vue-icons";
+import AppLayout from "@/Layouts/AppLayout.vue";
+import Navbar from "@/Components/Navbar.vue";
 </script>
 
 <template>
-    <Head title="Brevemente..." />
-    <!-- TODO: localization -->
-    <main class="min-h-screen bg-2023-bg">
-        <div
-            class="flex min-h-screen flex-col content-center items-center justify-evenly gap-14 py-16"
-        >
-            <img
-                id="svg-1"
-                class="absolute left-[7%] w-36 animate-2023-maintenance-jump opacity-50 max-ml:hidden"
-                src="images/cy-sinf.svg"
-                alt="Stylized SINF logo"
-            />
-            <img
-                id="svg-2"
-                class="absolute right-[7%] top-16 w-32 animate-2023-maintenance-jump opacity-50 max-ml:hidden"
-                src="images/rc-sinf.svg"
-                alt="Stylized SINF logo"
-            />
-            <img
-                id="svg-3"
-                class="absolute bottom-28 right-[15%] w-20 animate-2023-maintenance-jump opacity-50 max-ml:hidden"
-                src="images/oc-sinf.svg"
-                alt="Stylized SINF logo"
-            />
-
-            <div class="relative">
-                <img
-                    class="w-96 max-ml:w-72"
-                    src="images/sinf logo.png"
-                    alt="Stylized SINF logo"
-                />
-                <span
-                    class="margin-0 absolute -bottom-5 right-0 text-xl font-bold text-2023-teal"
-                    >2023</span
-                >
+    <AppLayout title="Home" :show-navbar="false">
+        <div class="min-h-screen flex flex-col">
```

Comment:
remove div
add flex and flex-col to applayout

Reply:
What do you mean by class prop? https://vuejs.org/guide/essentials/class-and-style Are we supposed to use this or just add the classes flex and flex-col to class